src/data/sql-quiz.ts

```diff
@@ -1033,6 +1033,19 @@ const sqlQuiz = [
     Explanation:
       "To select some particular date between one date and another, you can use the BETWEEN and AND keywords while specifying the dates. You can also use comparison operators such as greater than (>), less than (<), or equal(=).",
     Link: "https://www.freecodecamp.org/news/sql-date-function-query-timestamp-example-format/"
+  },
+  {
+    Question:
+      "Which of the following SQL queries correctly shows the number of unique subjects taken by a student?",
+    Answer: "SELECT COUNT(DISTINCT SUBJECT_ID)",
```

Comment:
I am not a SQL expert, but this looks like it is missing the from clause.

This should be valid

```suggestion
    Answer: "SELECT COUNT(DISTINCT SUBJECT_ID) FROM table_name;",
```